metadeploy/multisalesforce/views.py

```diff
@@ -55,7 +56,16 @@ def get_org_details(self, extra_data, token):
             version="44.0", org_id=org_id
         )
         resp = requests.get(org_url, headers=headers)
-        resp.raise_for_status()
+        if (
+            resp.status_code == 403
+            and json.loads(resp.text)[0]["errorCode"] == "API_DISABLED_FOR_ORG"
+        ):
+            raise SalesforcePermissionsError(
+                "Sorry, MetaDeploy requires an org with the API enabled."
+            )
+        else:
+            resp.raise_for_status()
+
         return resp.json()
```

Comment:
A small tweak to parse the json upfront regardless of whether we're handling an error or not, and to be more defensive in case the response isn't in the expected format.

```suggestion
        resp_json = resp.json()
        if (
            resp.status_code == 403
            and len(resp_json) and resp_json[0]["errorCode"] == "API_DISABLED_FOR_ORG"
        ):
            raise SalesforcePermissionsError(
                "Sorry, MetaDeploy requires an org with the API enabled."
            )
        else:
            resp.raise_for_status()
        return resp_json
```